packages/core/src/Card/Card.tsx

```diff
@@ -46,7 +46,7 @@ export const HvCard = (props: HvCardProps) => {
     selectable = false,
     selected = false,
     statusColor = "sema0",
-    bgcolor,
+    bgcolor = "atmo1",
```

Comment:
IIRC, we discussed making this change in the past, but considered it a breaking change as it affects many `HvCard` that don't want to have a background